srv/components/provisioner/scripts/install.sh

```diff
@@ -260,7 +261,7 @@ config_local_salt()
 setup_repos_hosted()
 {
     repo=$1
-    for repo_file in $(grep -lE "cortx-storage.colo.seagate.com|file://" /etc/yum.repos.d/*.repo); do
+    for repo_file in $(grep -l "baseurl=file:///" /etc/yum.repos.d/*.repo); do
```

Comment:
will this work for http based url?

Reply:
yes as tested with local hosted rpms using nginx